### PR TITLE
TAPE rename

### PR DESCRIPTION
**Pull request by dougbrn**
lsstseries -> TAPE

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ In short term we are working on two main goals of the project:
 
 This is a LINCC Frameworks project - find more information about LINCC Frameworks [here](https://www.lsstcorporation.org/lincc/frameworks)
 
-To learn about the usage of the package, consult the [Documentation](https://lsstseries.readthedocs.io/en/latest/index.html).
+To learn about the usage of the package, consult the [Documentation](https://tape.readthedocs.io/en/latest/index.html).
```
Comment (jeremykubica) [inline README.md:18]: Have you already reserved this? I figured if tape.readthedocs was taken we'd need to do something like LINCC-tape.readthedocs for the webpage names.

File: docs/notebooks/working_with_the_ensemble.ipynb [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+    "# Working with the TAPE `Ensemble` object"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When working with many lightcurves, the lsstseries `Ensemble` object serves as a singular interface for storing, filtering, and analyzing timeseries data. \n",
+    "When working with many lightcurves, the tape `Ensemble` object serves as a singular interface for storing, filtering, and analyzing timeseries data. \n",
```
Comment (jeremykubica) [inline docs/notebooks/working_with_the_ensemble.ipynb:16]: tape is all caps in the rest of the notebooks. Maybe do that here for consistency.

File: docs/notebooks/working_with_the_ensemble.ipynb [hunk 3]
```diff
@@ -343,19 +349,20 @@
...
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Using a Custom Analysis Function\n",
-    "The analysis functions contained in lsstseries are meant to provide a collection of performant, on-hand routines for common timeseries use cases. However, lsstseries is also equipped to handle externally defined functions. Let's walk through a short example of defining a simple custom function and applying it through `Ensemble.batch`.\n",
+    "The analysis functions contained in tape are meant to provide a collection of performant, on-hand routines for common timeseries use cases. However, tape is also equipped to handle externally defined functions. Let's walk through a short example of defining a simple custom function and applying it through `Ensemble.batch`.\n",
```
Comment (jeremykubica) [inline docs/notebooks/working_with_the_ensemble.ipynb:365]: capitalize tape

File: docs/notebooks/working_with_the_timeseries.ipynb [hunk 4]
```diff
@@ -5,9 +5,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Working with the lsstseries `TimeSeries` object\n",
+    "# Working with the TAPE `TimeSeries` object\n",
     "\n",
-    "By contrast to the `Ensemble`, which operates on many lightcurves, the lsstseries `TimeSeries` object operates on a single lightcurve. \n",
+    "By contrast to the `Ensemble`, which operates on many lightcurves, the tape `TimeSeries` object operates on a single lightcurve. \n",
```
Comment (jeremykubica) [inline docs/notebooks/working_with_the_timeseries.ipynb:10]: TAPE

File: src/tape/ensemble.py [hunk 5]
```diff
@@ -573,19 +573,19 @@ def bin_sources(
...
+        """Run a function from tape.TimeSeries on the available ids
 
         Parameters
         ----------
         func : `function`
             A function to apply to all objects in the ensemble
         *args:
             Denotes the ensemble columns to use as inputs for a function,
-            order must be correct for function. If passing a lsstseries
+            order must be correct for function. If passing a tape
```
Comment (jeremykubica) [inline src/tape/ensemble.py:584]: TAPE

File: src/tape/ensemble.py [hunk 6]
```diff
@@ -573,19 +573,19 @@ def bin_sources(
...
             function, these are populated automatically.
         meta : `pd.Series`, `pd.DataFrame`, `dict`, or `tuple-like`
             Dask's meta parameter, which lays down the expected structure of
-            the results. Overridden by lsstseries for lsstseries
+            the results. Overridden by tape for tape
```
Comment (jeremykubica) [inline src/tape/ensemble.py:588]: TAPE

**Review by jeremykubica — APPROVED** [review]
Approving. Only very small style comments.

File: README.md (hunk 1 above)
Comment (dougbrn) [reply README.md:18]: Not reserved, but looks free. We'll see if rtd runs any issues with the first build from main

**Comment by dougbrn** [comment]
tape->TAPE suggestions added in the latest commit

**Pull request merged**